### DISCUSSION TITLE
Margin adjustment and printing options

### DISCUSSION THREAD

**Discussion by Mailliwivad** [sumpfork/dominiontabs#493] [Q&A] [answered]
I am looking to get some printed at a print shop on card stock. They want me to try to provide a file without too much white space for their 12x18 inch sheets. Is there a way to adjust the the margins on the output file and/or the size of the output paper other than the set options?

Also is there a way to change the printout so that the back side of the dividers do not flip? (The print shop does this manually with the commercial printer apparently).

Thanks for any help

**Comment by sumpfork** [comment, marked as answer]
There are [quite a few options to adjust margins and print sizes](https://github.com/sumpfork/dominiontabs/wiki/Documentation-%28Script-Options%29). Try things like `--minmargin` together with diivider size and page size adjustments.

I don't think I understand the second requirement. How is the printer going to flip the tab outline without flipping the text and other contents?

**Reply by Mailliwivad** [reply]
Thanks so much for the quick response. I had only been using the web version, but I will try to figure out how to install the package and run it locally.

As for the second requirement, I am unsure why they need it that way or how their printer works. I believe they set everything up in some program beforehand, but I reiterated that every second page needs to line up with the page before it. I think they can work with the output if not possible, but they were asking me for it not to flip, so I figured I'd ask.

Thanks again.

**Reply by sumpfork** [reply]
Yeah, this'd require a code change. And a better spec, like, should the whole page be flipped or just the outlines?
